src/python/tools/configuration/marius_config.py

```diff
@@ -387,7 +387,7 @@ def merge(self, input_config: DictConfig):
 
 @dataclass
 class DatasetConfig:
-    base_directory: str = MISSING
+    dataset_dir: str = MISSING
     num_edges: int = MISSING
```

Comment:
Can set the dataset info to -1 instead of MISSING if that is why the additional code is needed here: https://github.com/marius-team/marius/pull/92/files#diff-8bcac21e795733ad91545427dc9e48a66e5f04562e5be8d911c8f951d60cba3fR26

Reply:
umm, the purpose of that additional code is different. explained below.